src/library/Uno.Toolkit.Material/Styles/Controls/v2/Divider.xaml

```diff
@@ -21,12 +35,11 @@
 							<RowDefinition Height="Auto" />
 						</Grid.RowDefinitions>
 
-						<Rectangle Fill="{TemplateBinding Foreground}"
-								   Height="{StaticResource MaterialDividerHeight}" />
+						<Rectangle Fill="{TemplateBinding Foreground}" Height="{ThemeResource MaterialDividerHeight}" />
 
 						<TextBlock Text="{TemplateBinding SubHeader}"
 								   Visibility="{Binding SubHeader, RelativeSource={RelativeSource Mode=TemplatedParent}, Converter={StaticResource MaterialEmptyToCollapsed}, FallbackValue=Collapsed}"
-								   Style="{StaticResource BodySmall}"
+								   Style="{ThemeResource DividerSubHeaderStyle}"
 								   Foreground="{TemplateBinding SubHeaderForeground}"
 								   Margin="0,4,0,0"
```

Comment:
Margin="0,4,0,0"

Can we have this hardcoded margin part of the lightweight styling values please